git_machete/client.py

```diff
@@ -1851,7 +1852,11 @@ def checkout_github_prs(self,
         remote, (org, repo) = self.__derive_remote_and_github_org_and_repo()
         current_user: Optional[str] = git_machete.github.derive_current_user_login()
         if not current_user and my_opened_prs:
-            warn("Could not determine current user name, please check your token.")
+            if called_function:
+                warn("Could not determine current user name, please check your token.")
```

Comment:
Can be done on separate ticket &mdash; would be good to display a more verbose message about what token is missing, or at least redirect the user to `help`

Reply:
https://github.com/VirtusLab/git-machete/issues/468